### DISCUSSION TITLE
Move Panel to the right side in the builder

### DISCUSSION THREAD

**Discussion by lifeofsa** [GrapesJS/grapesjs#5429] [Q&A]
How do I move the Panel to right side with buttons?

I want to move the PageManager and BlockManager on the right side of the builder with their respective buttons.. Kindly someone help please.

![GRAPESJS](https://github.com/GrapesJS/grapesjs/assets/83239811/c8d7b1f2-8373-4eea-841a-11e7380a3464)

**Comment by julia-alberici** [comment]
I don't believe you'll be able to achieve that without building a custom UI, I recommend following the getting started section guide from the docs to learn how to build a custom UI
